### PR TITLE
Remote access with XML catalog as alternative for downloading DSTs

### PR DESCRIPTION
**Pull request by pieterdavid**
Slightly more advanced than downloading files, but I think it is good to know that that can relatively easily be avoided

### CODE REVIEW

**Comment by betatim** [comment]
Thanks for this!

What do others think about adding in the appropriate `code/` directory an example that uses the XML catalog instead of the local file?

**Comment by ibab** [comment]
:+1: on `code/` example

**Comment by pieterdavid** [comment]
Something like this?

File: 04-files-from-grid.md [hunk 1]
```diff
@@ -46,6 +46,22 @@ Again this will take a while but afterwards you should have a file called `00035
 
 Since those tuples tend to be quite large, you might want to use your AFS work directory instead of your AFS user directory to store files.
 
+> ## Alternative: read the remote files instead of downloading them {.callout}
```
Comment (betatim) [inline 04-files-from-grid.md]: "read the remote files" -> "read files remotely instead of"

**Comment by betatim** [comment]
Looks good to me (just fix the little english language thing), then we can merge.

We send :cake: from #MozFest. Thanks

File: 04-files-from-grid.md [hunk 2]
```diff
@@ -46,6 +46,22 @@ Again this will take a while but afterwards you should have a file called `00035
 
 Since those tuples tend to be quite large, you might want to use your AFS work directory instead of your AFS user directory to store files.
 
+> ## Alternative: read files remotely instead of downloading them {.callout}
+> To avoid filling up your AFS quota with dst files, you can also pass Gaudi an XML catalog such that it can access them remotely.
```
Comment (alexpearce) [inline 04-files-from-grid.md]: Uppercase DST.

**Comment by alexpearce** [comment]
This is great, thank you! And sorry so the enormous delay 🙇 There's just one tiny change, and then I'd so it's ready to merge.

**Comment by pieterdavid** [comment]
In the mean time I can't even test this anymore, so I just changed the one word and hope the rest is still fine :-)

**Comment by alexpearce** [comment]
Let's hope so 😉

**Pull request merged**